README.md

```diff
@@ -72,15 +68,28 @@ By granting the app access, you are providing the following authorizations to yo
 **Subscribe to an Organization or a Repository**
 On repositories, the app notifies of `open`, `close`, and `re-open` events on pull requests and issues in repositories you've subscribed to. It also notifies of any `push` directly to the repository's default branch as well as `comments` on issues and pull requests.
 
-#### Types of Public Link Unfurls
-When a user posts a GitHub link in Slack for a public repository, the app is designed to unfurl **issues and pull requests**, directly linked **comments**, code **blobs** with line numbers, as well as **organizations, repositories, and users**.
+#### Link previews
+<!-- preserve old anchor in case other docs link there -->
+<a name="types-of-public-link-unfurls"></a>
+
+When a user posts a GitHub link to **issues and pull requests**, directly linked **comments**, code **blobs** with line numbers, as well as **organizations, repositories, and users** in Slack, a preview of the link will be shown.
 
```

Comment:
Suggestion: For this feature to work you need to invite the GitHub integration to the relevant Slack channels using `/invite @github`